tesseract_motion_planners/trajopt/include/tesseract_motion_planners/trajopt/trajopt_utils.h

```diff
@@ -36,21 +36,27 @@ TESSERACT_COMMON_IGNORE_WARNINGS_POP
 
 namespace tesseract_planning
 {
-trajopt::TermInfo::Ptr createCartesianWaypointTermInfo(int index,
-                                                       const std::string& working_frame,
-                                                       const Eigen::Isometry3d& c_wp,
-                                                       const std::string& tcp_frame,
-                                                       const Eigen::Isometry3d& tcp_offset,
-                                                       const Eigen::VectorXd& coeffs,
-                                                       trajopt::TermType type);
-
-trajopt::TermInfo::Ptr createDynamicCartesianWaypointTermInfo(int index,
-                                                              const std::string& working_frame,
-                                                              const Eigen::Isometry3d& c_wp,
-                                                              const std::string& tcp_frame,
-                                                              const Eigen::Isometry3d& tcp_offset,
-                                                              const Eigen::VectorXd& coeffs,
-                                                              trajopt::TermType type);
+trajopt::TermInfo::Ptr
+createCartesianWaypointTermInfo(int index,
+                                const std::string& working_frame,
+                                const Eigen::Isometry3d& c_wp,
+                                const std::string& tcp_frame,
+                                const Eigen::Isometry3d& tcp_offset,
+                                const Eigen::VectorXd& coeffs,
+                                trajopt::TermType type,
+                                const Eigen::VectorXd& lower_tolerance = Eigen::VectorXd::Zero(6),
+                                const Eigen::VectorXd& upper_tolerance = Eigen::VectorXd::Zero(6));
+
```

Comment:
I think the default should be empty size.

Reply:
Now initializing with
```C++
const Eigen::VectorXd& lower_tolerance = Eigen::VectorXd()
```